src/app/core/callendar/transformers/default-data-transformer.ts

```diff
@@ -19,7 +20,8 @@ export class DefaultDataTransformer  implements IcalendarTransformer<SessionView
             start: moment(session.start),
             end: moment(moment(session.start).add(moment.duration(session.duration))),
             id: session.id,
-            hearingParts: session.hearingParts
+            hearingParts: session.hearingParts,
+            color: EventsColorsForCasetype[caseType] || 'gray'
```

Comment:
how about change 'gray' to `EventsColorsForCasetype['No Case type']` (or even remove spaces and use `EventsColorsForCasetype.NoCaseType`